sdk/python/feast/driver_test_data.py

```diff
@@ -198,3 +198,41 @@ def create_customer_daily_profile_df(customers, start_date, end_date) -> pd.Data
     # TODO: Remove created timestamp in order to test whether its really optional
     df_all_customers["created"] = pd.to_datetime(pd.Timestamp.now(tz=None).round("ms"))
     return df_all_customers
+
+
+def create_global_daily_stats_df(start_date, end_date) -> pd.DataFrame:
```

Comment:
Do you mean that instead of `create_global_daily_stats_df` creating its own dataset of columns `event_timestamp`, `num_rides`, `avg_ride_length`, and `created`, that I could use one of the others like `create_driver_hourly_stats_df` and just remove `driver_id`?

Wouldn’t that be confusing when it comes to using the same feature names? (and could break some tests that parametrize with `full_feature_names=False`). I think it could make sense to just have the separate, new entityless dataset to represent an example of entityless features.

Reply:
Fair enough. Lets stick to this approach.

Reply:
@codyjlin and I talked offline - fine to stick with this approach as well.